src/main/java/net/dzikoysk/funnyguilds/element/gui/GuiActionHandler.java

```diff
@@ -1,53 +1,34 @@
 package net.dzikoysk.funnyguilds.element.gui;
 
-import org.bukkit.event.Event;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
-import org.bukkit.event.inventory.InventoryClickEvent;
-import org.bukkit.event.inventory.InventoryCloseEvent;
-import org.bukkit.event.inventory.InventoryInteractEvent;
-import org.bukkit.event.inventory.InventoryOpenEvent;
-import org.bukkit.event.inventory.InventoryType;
+import org.bukkit.event.inventory.*;
```

Comment:
There isn't any project code style, but I think the majority would agree that we should avoid wildcard imports.